bin/README.md

```diff
@@ -0,0 +1,26 @@
+## Bin
+
+This directory contains a number of scripts for running pathway parameter advising, updating reference pathways, and installing PGD.
+
+Most of these scripts must be run inside the `bin` directory. 
+
+## Files
+
+`bin/runPPA.sh` runs pathway parameter advising on any set of sif or edgelist networks.
+It will attempt to install the pgd library and setup the reactome pathway graphlets automatically. 
```

Comment:
It now requires PGD to be pre-installed.

Reply:
I removed this mention, requiring the installation of PGD as an argument I think gets this across now.